src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -324,7 +324,7 @@ std::unique_ptr<model_backend> tf_compute_context::create_activity_classifier(
 
     // Make an instance of python object
     pybind11::object activity_classifier = tf_ac_backend.attr(
-        "ActivityTensorFlowModel")(weights, n, c_in, c_out, pw, w_out);
+        "ActivityTensorFlowModel")(weights, n, c_in, c_out, pw, w_out, config);
```

Comment:
This `config` dictionary is a vestige of the original TCMPS interface for binding a C backend to the Python frontend. Could we just add normal Pythonic arguments to `ActivityTensorFlowModel`?

Reply:
Yes